partner_firstname/models/res_partner.py

```diff
@@ -126,18 +142,6 @@ def _inverse_name(self):
             record.lastname = parts["lastname"]
             record.firstname = parts["firstname"]
 
-    @api.constrains("firstname", "lastname")
-    def _check_firstname_lastname(self):
-        """Ensure at least one name is set."""
-        for record in self:
-            if all(
-                (
-                    record.type == "contact" or record.is_company,
-                    not (record.firstname or record.lastname),
-                )
-            ):
-                raise exceptions.EmptyNamesError(record, self.env)
-
```

Comment:
This should not be removed.

Reply:
Hi @mt-software-de. I don't get your point. This check is still present in the mixin. 

https://github.com/OCA/partner-contact/pull/2174/files?file-filters%5B%5D=.py&show-viewed-files=true#diff-41631818bda17e301e01286121510fd1d098583645448aa21b61f95a76e66f4aR160